manifests/core-services.yaml

```
@@ -19,6 +19,8 @@ spec:
     version: 2.1.9
     namespace: services
     values:
+      global:
+        appVersion: 1.58.1
```

We should also specify the test image version to be in alignment. 
```suggestion
        appVersion: 1.58.1
        testVersion: 1.58.1
```